src/pyramid/config/routes.py

```diff
@@ -220,25 +220,23 @@ def add_route(
         header
 
           This value can be a string or an iterable of strings for HTTP
```

Comment:
Because "value" is used in the docs below, although in a different context, I would like to reserve its use for the explanation to avoid potential confusion.

```suggestion
          This argument can be a string or an iterable of strings for HTTP
```